src/assignment/hooks/useGetAssignmentsEditStatuses.tsx

```diff
@@ -4,15 +4,13 @@ import { AssignmentService } from '../assignment.service';
 
 export const useGetAssignmentsEditStatuses = (
 	assignmentsIds: string[],
-	enabled: boolean,
-	interval: number,
 	options: {
 		enabled: boolean;
 		refetchInterval: number;
-		refetchIntervalInBackground: boolean;
+		refetchIntervalInBackground?: boolean;
 	} = {
-		enabled: enabled || false,
-		refetchInterval: interval || 0,
+		enabled: false,
+		refetchInterval: 0,
 		refetchIntervalInBackground: true,
```

Comment:
use the library's default value for these:
https://github.com/TanStack/query/blob/9b48048a61bfc50483e4a7e16fa28ac23626896d/packages/query-core/src/types.ts#L138-L154

otherwise you'll refetch constantly
```suggestion
		refetchInterval: false,
		refetchIntervalInBackground: true,
```

Reply:
refetchInterval key expects only numbers 
<img width="1012" alt="image" src="https://github.com/viaacode/avo2-client/assets/113341869/82b78633-40aa-414f-9cca-a61c77ec7207">